docs/wiki/Interoperability - Bicep to ARM conversion.md

```diff
@@ -1,6 +1,10 @@
-Bicep is a relatively new Domain-specific language (DSL) and - at the time of writing - not yet in the state of General Availability (GA). For this reason, some people may want to wait for Bicep's _General Availability_ and prefer to use ARM/JSON for the time being.
+
+Bicep is [ready for production use](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/frequently-asked-questions#is-this-ready-for-production-use) starting with version 0.3. It is supported by Microsoft support plans and has parity with what can be accomplished with ARM Templates.
+
+However, Bicep is a relatively new Domain-specific language (DSL) and - at the time of writing - not yet in version 1.0. For this reason, some users may want to wait before adopting Bicep in their environment and prefer to use ARM/JSON for the time being.
```

Comment:
Should we entirely remove this small paragraph? I don't think it adds value, but more of an indirect warning.

Reply:
We should however still somehow lead over to the ARM converter

Reply:
I'm with you, but if we want to continue supporting Bicep conversion to ARM/JSON we need to justify it. Thoughts?

Reply:
Maybe something like: for users who prefer to use ARM templates over bicep, CARML provides a utility . . . .

Reply:
updated